threescale_api/resources.py

```diff
@@ -865,11 +865,9 @@ def plans_url(self) -> str:
     def service(self) -> 'Service':
         return self.parent
 
-    @property
```

Comment:
also this looks like breaking change, despite to fact that it doesnt make sense to have property function with optional args -> certainly there is something wrong original code

Reply:
I don't think this is a breaking change because there is no test setting up directly limits and always it is called like this 
```
limits = app_plan.limits(metric)._something_
```
So I think removing `@property` is OK.